lib/index.js

```diff
@@ -128,7 +132,13 @@ AxeBuilder.prototype.analyze = function(callback) {
     config = this._config,
     source = this._source;
 
-  return new Promise(function(resolve) {
+  // Check if the provided `callback` uses the old argument signature (an arity of 1). If it does, provide a helpful deprecation warning.
+  var isOldAPI = callback && callback.length === 1;
+  if (isOldAPI) {
+    deprecate('When using the callback API, you must handle errors');
```

Comment:
I suggest: "Error must be handled as the first argument of axe.analyze. See: https://github.com/dequelabs/axe-webdriverjs/pull/83"